crates/subspace-farmer/Cargo.toml

```diff
@@ -37,11 +37,12 @@ ss58-registry = "1.10.0"
 subspace-archiving = { version = "0.1.0", path = "../subspace-archiving" }
 subspace-solving = { version = "0.1.0", path = "../subspace-solving" }
 subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives" }
+subspace-networking = { version = "0.1.0", path = "../subspace-networking" }
 subspace-rpc-primitives = { version = "0.1.0", path = "../subspace-rpc-primitives" }
 thiserror = "1.0.30"
 tiny-bip39 = "0.8.2"
 tokio = { version = "1.16.1", features = ["macros", "rt-multi-thread"] }
-zeroize = "1.5.2"
+zeroize = "1.4.3"
```

Comment:
Is there a reason for downgrade? `Cargo.lock` is not changed though

Reply:
Unfortunately yes, `chacha20poly1305` crate has upper bound on `zeroize` dependency that is currently set to `>=1, <1.5`, by having version `1.5.2` here (that is within the same major version) Cargo is unable to resolve dependency as it doesn't keep 2 copies of the same major version of the crate:
```
error: failed to select a version for `zeroize`.
    ... required by package `chacha20poly1305 v0.9.0`
    ... which satisfies dependency `chacha20poly1305 = "^0.9"` of package `snow v0.9.0`
    ... which satisfies dependency `snow = "^0.9.0"` of package `libp2p-noise v0.35.0 (https://github.com/libp2p/rust-libp2p#b39770b8)`
    ... which satisfies git dependency `libp2p-noise` of package `libp2p v0.43.0 (https://github.com/libp2p/rust-libp2p#b39770b8)`
    ... which satisfies git dependency `libp2p` of package `subspace-networking v0.1.0 (/web/subspace/subspace/crates/subspace-networking)`
    ... which satisfies path dependency `subspace-networking` (locked to 0.1.0) of package `subspace-farmer v0.3.0 (/web/subspace/subspace/crates/subspace-farmer)`
versions that meet the requirements `>=1, <1.5` are: 1.4.3, 1.4.2, 1.4.1, 1.4.0, 1.3.0, 1.2.0, 1.1.1, 1.1.0, 1.0.0

all possible versions conflict with previously selected packages.

  previously selected package `zeroize v1.5.2`
    ... which satisfies dependency `zeroize = "^1.4.3"` of package `sc-network v0.10.0-dev (https://github.com/paritytech/substrate?rev=2c4549689dbb86c23725dac2f82af35faa07c9f6#2c454968)`
    ... which satisfies git dependency `sc-network` (locked to 0.10.0-dev) of package `cirrus-client-executor-gossip v0.1.0 (/web/subspace/subspace/cumulus/client/executor-gossip)`
    ... which satisfies path dependency `cirrus-client-executor-gossip` (locked to 0.1.0) of package `cirrus-client-executor v0.1.0 (/web/subspace/subspace/cumulus/client/cirrus-executor)`
    ... which satisfies path dependency `cirrus-client-executor` (locked to 0.1.0) of package `cirrus-client-service v0.1.0 (/web/subspace/subspace/cumulus/client/cirrus-service)`
    ... which satisfies path dependency `cirrus-client-service` (locked to 0.1.0) of package `parachain-template-node v0.1.0 (/web/subspace/subspace/cumulus/parachain-template/node)`

failed to select a version for `zeroize` which could resolve this conflict
```